workspace/package.go

```diff
@@ -151,6 +152,16 @@ func (p Package) ExecuteToWriter(runtimeCtx *runtime.Runtime, output io.Writer)
 	return nil
 }
 
+func (p Package) checkMounts(cd narwhal.ContainerDefinition, srcDir string) {
```

Comment:
nit: `ContainerDefinition` is a large struct and should generally be passed around by pointer. I know it isn't being passed by value everywhere, but I'm trying to reduce the places where it is.

```suggestion
func (p Package) checkMounts(cd *narwhal.ContainerDefinition, srcDir string) {
```

See https://github.com/golang/go/wiki/CodeReviewComments#receiver-type for a longer discussion.

Reply:
Thanks :+1: